guides/common/modules/proc_altering-the-privilege-elevation-method.adoc

```diff
@@ -0,0 +1,22 @@
+[id="altering-the-privilege-elevation-method_{context}"]
+= Altering the Privilege Elevation Method
+
+By default, push-based remote execution uses `sudo` to switch from the SSH user to the effective user that executes the script on your host.
+In some situations, you may require to use another method, such as `su` or `dzdo`.
+You can globally configure an alternative method to switch to the effective user in your {Project} settings.
+
+.Prerequisite
+* If you want to use `dzdo` for Ansible jobs, you have to install the `community.general` Ansible collection, which contains the *dzdo* Ansible plug-in.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/solutions/7025758[Running Ansible job on hosts registered with Red Hat Satellite 6 fails with error could not find matching plugin: 'dzdo'] in the _Red{nbsp}Hat Knowledgebase_.
```

Comment:
I don't think you should link to the solution. If a user lands on this module, they will not encounter the error described in the solution after following your procedure.

If anything, I believe the solution should link to your new module.

Reply:
Probably just the Ansible collection installation link will suffice.